app/views/vm_common/_reconfigure.html.haml

```diff
@@ -343,8 +346,25 @@
                                                               "ng-click" => "vm.cancelAddRemoveNetworkAdapter(networkAdapter)"}= _('Cancel Add')
         %tr{"ng-if"       => "vm.reconfigureModel.showDropDownNetwork",
             "ng-form"     => "rowForm"}
-          %td
-          %td
+          %td{"ng-if" => "!vm.isVmwareCloud()"}
+          %td{"ng-if" => "vm.isVmwareCloud()"}
+            = select_tag('adapterIdx',
+                       options_for_select([["<#{_('Choose')}>", '']] + @avail_adapter_names, :disabled => ["<#{_('Choose')}>", nil]),
+                       "ng-model"                    => "vm.reconfigureModel.name",
+                       "ng-change"                   => "",
+                       "data-width"                  => "auto",
+                       "required"                    => "true",
+                       "selectpicker-for-select-tag" => "")
+          %td{"ng-if" => "vm.isVmwareCloud()"}
+            %select{'name'                    => 'adapterNetwork',
+                'ng-model'                    => 'vm.reconfigureModel.adapterNetwork',
+                'ng-options'                  => 'network for network in vm.reconfigureModel.availableAdapterNetworks',
+                'pf-select'                   => true,
+                'required'                    => false,
```

Comment:
Depending on the haml implementation `%x{ :required => false }` is either `<x></x>` or `<x required="false"></x>`. The string `"false"` is still true :).

I suggest you just remove the line.